### PR TITLE
Gradle 7.0.2 and GitHub Actions

### PR DESCRIPTION
**Pull request by tofikarz**
This PR:

+ Bumps loom from 0.6-SNAPSHOT to 0.8-SNAPSHOT 

+ Adds GitHub Actions to validate Gradle Wrapper

+ Updates Gradle to 7.0.2

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: LazyDFU build script
```
Comment (astei) [inline .github/workflows/build.yml:1]: We have a long-established job I set up on the [Velocity Jenkins](https://ci.velocitypowered.com/job/lazydfu/). GitHub Actions isn't necessary.

Comment (tofikarz) [reply .github/workflows/build.yml]: I left Gradle Wrapper Validation for security reasons if someone wanted to replace it with malware

**Comment by DragonEggBedrockBreaking** [comment]
this is more important now that minecraft uses java 16

**Comment by astei** [comment]
I'm reverting this, it's broken.

@DragonEggBedrockBreaking LazyDFU is intended to work with a wide range of Minecraft versions. _I do not care about you being able to build on a newer JDK_. Java 8 and 11 are LTS versions for quite a while longer, and it's stupid to drop support for ensuring older versions of Minecraft can use LazyDFU.

**Comment by DragonEggBedrockBreaking** [comment]
> _I do not care about you being able to build on a newer JDK_.

Btw, I was referring to Minecraft moving to JDK 16 in 21w19a, not specific people using JDK 16 on older versions.

**Comment by DragonEggBedrockBreaking** [comment]
And another comment, loom 0.6 can still be used to build on jdk 16 on 1.16 or below, assuming grade 7, and you only reverted the loom thing. Loom 0.8 is only required for 1.17.

**Pull request merged**